development/cpp/configuring_an_ide.rst

```diff
@@ -296,3 +297,22 @@ To create them:
 - You can now start Debug process again to test that everything works.
 
 - If the build phase fails, check the console for hints. On Linux it's most likely that some dependencies are missing. Check :ref:`Compiling for X11 (Linux, \*BSD) <doc_compiling_for_x11>`
+
+
+.. _doc_configuring_an_ide_android_studio:
+
+Android Studio
+-----------------
+
+`Android Studio <https://developer.android.com/studio>`_ is a `Jetbrain <https://www.jetbrains.com/>`_ IDE for Android development. It has a features rich editor which supports Java and C/C++, so it can be used for development of the Godot core engine, and Android platform codebases.
```

Comment:
Not a native speaker myself but usually I see it written as "feature-rich" instead of "features rich".